src/core/commandVersionUpgrades/upgradeV1ToV2/commandV1.types.ts

```diff
@@ -84,6 +84,7 @@ type SelectionType =
   | "paragraph"
   | "document"
   | "nonWhitespaceSequence"
+  | "nonWhitespaceOrQuoteSequence"
```

Comment:
These files should never change.  Or is that necessary for some old tests you have?  I guess that's ok? cc/ @AndreasArvidsson

Reply:
Agreed we should not change this file.

Reply:
My bad, I was looking for anywhere that nonWhitespaceSequence appeared.